workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -96,3 +100,16 @@ def write_chunks(config: UserConfig):
     chunks = get_chunks(diagnostic_file_configs)
     with open("chunks.yaml", "w") as f:
         yaml.safe_dump(chunks, f)
+
+
+def _standardize_prephysics_config(config) -> PrephysicsConfig:
+    # allows for backwards compatibility for old config format where
+    # only a single entry for prephysics was allowed
+    if "prephysics" not in config:
+        return config
+    elif isinstance(config["prephysics"], List):
+        return config
+    else:
+        raw_prephysics_config = config["prephysics"]
+        config["prephysics"] = [raw_prephysics_config]
+        return config
```

Comment:
As another option, did you try type-hinting `config.prephysics` as

```
Prephysics = Union[PrescriberConfig, MachineLearningConfig]
Optional[Union[List[Prephysics], Prephysics]]
```

Reply:
I think I'll just make this non-backwards-compatible instead of moving the backwards compatible code to the loop.  If someone tries to use an old config this will raise an error before starting the run that is easy to fix in the config.

Reply:
Agree, it's not like locking out a saved model, it's just a change in the config